text_handler.py

```diff
@@ -26,42 +29,58 @@ async def __handleYoutubeDownload(self, message: types.Message):
+                ydl.download([message.text])
+                duration = format(info['duration'])
+                extractor = format(info['extractor'])
+                title = format(info['title'])
+                audio_ext = format(info['ext'])
+                webpage_id = format(info['id'])
+                webpage_url = format(info['webpage_url'])
+                uploader = format(info['uploader'])
+                thumbnail_url = format(info['thumbnail'])
+                thumbnail_urlpath = urlparse(thumbnail_url).path
+                thumbnail_filename = os.path.basename(thumbnail_urlpath)
+                thumbnail_ext = thumbnail_filename.split(".")[-1]
+                base_filename = "downloads/" + extractor + '_' + \
+                    webpage_id + '.'
+                output_audiofile = base_filename + audio_ext
+                original_thumbnail = base_filename + thumbnail_ext
+                output_thumbnail = "downloads/square_thumbnail_" + \
+                    webpage_id + ".jpg"
+                self.__make_squarethumb(original_thumbnail, output_thumbnail)
                 with open(output_audiofile, "rb") as audio, \
                      open(output_thumbnail, "rb") as thumb:
                     await message.reply_audio(audio,
-                                              caption="<a href=\"" + video_url +
-                                                      "\">" + title + "</a>",
+                                              caption="<b><a href=\"" +
+                                              webpage_url + "\">" +
+                                              title + "</a></b>",
                                               parse_mode=ParseMode.HTML,
-                                              duration=length,
-                                              performer=author,
+                                              duration=duration,
+                                              performer=uploader,
                                               title=title,
                                               thumb=thumb)
                 os.remove(output_audiofile)
+                os.remove(original_thumbnail)
                 os.remove(output_thumbnail)
             else:
-                await message.reply("`No downloads for 10min\\+ audio " +
-                                    "or file size greater than 50M`",
-                                    parse_mode=ParseMode.MARKDOWN_V2)
+                await message.reply("The audio file has the size of " +
+                                    str(audio_filesize) + " bytes. " +
+                                    "Bots can currently send files of " +
+                                    "any type of only up to 50 MB " +
+                                    "in size (52428800 bytes)")
         except Exception as e:
             await message.reply("I've tried downloading this video but " +
```

Comment:
i think we should replace this message reply with something like 
```
        except Exception as e:
            logger.error("text handler error: " + str(e))
```

Reply:
Just noticed that I did this incorrectly, anyway this will be fixed in later commit.

Reply:
Review the last commit